system_tests/pubsub.py

```diff
@@ -80,8 +82,13 @@ def test_list_topics(self):
             self.to_delete.append(topic)
 
         # Retrieve the topics.
-        all_topics, _ = Config.CLIENT.list_topics()
-        created = [topic for topic in all_topics
+        def _all_created(result):
+            return len(result[0]) == len(before) + len(topics_to_create)
```

Comment:
This predicate doesn't quite cover it, i.e. if other actors updated the list of topics. We essentially want to check all of the names. Also, why only use `result[0]`? Can you instead tuple-unpack so it's clear what the values are?